crates/ruma-common/src/events/poll/end.rs

```diff
@@ -105,3 +189,59 @@ impl Deref for PollResultsContentBlock {
         &self.0
     }
 }
+
+/// Generate the current results with the given poll and responses.
+///
+/// If the `end_timestamp` is provided, any response with an `origin_server_ts` after that timestamp
+/// is ignored. If it is not provided, `MilliSecondsSinceUnixEpoch::now()` will be used instead.
+///
+/// This method will handle invalid responses, or several response from the same user so all
+/// responses to the poll should be provided.
+///
+/// Returns a map of answer ID to a set of user IDs that voted for them. When using `.iter()` or
+/// `.into_iter()` on the map, the results are sorted from the highest number of votes to the
+/// lowest.
+pub fn compute_poll_results<'a>(
```

Comment:
This also feels weird to have in the end module specifically. Maybe it should be in the parent?

Reply:
Alright